### DISCUSSION TITLE
User for sshd

### DISCUSSION THREAD

**Discussion by F4u5t** [termux/termux-app#3406] [General]
Im not sure I understand how to correctly ssh into Termux, with the user changing on every boot.
I currently have to run:
whoami
then passwd to set the password before i can ssh...
is there a better way?

**Comment by twaik** [comment]
sshd of termux ignores username so you can login with any username you want.

**Reply by agnostic-apollo** [reply]
Don't even need to prefix the `username@` before ip and android doesn't change app username on every boot and only on app reinstalls.

**Reply by F4u5t** [reply]
> sshd of termux ignores username so you can login with any username you want.

well heck, never tried that ! thanks
